// sitemap/dist/index.js
"use strict";
function __export(m) {
    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
}
Object.defineProperty(exports, "__esModule", { value: true });
/*!
 * Sitemap
 * Copyright(c) 2011 Eugene Kalinin
 * MIT Licensed
 */
var sitemap_item_stream_1 = require("./lib/sitemap-item-stream");
exports.SitemapItemStream = sitemap_item_stream_1.SitemapItemStream;
var sitemap_index_stream_1 = require("./lib/sitemap-index-stream");
exports.IndexTagNames = sitemap_index_stream_1.IndexTagNames;
exports.SitemapIndexStream = sitemap_index_stream_1.SitemapIndexStream;
exports.createSitemapsAndIndex = sitemap_index_stream_1.createSitemapsAndIndex;
exports.SitemapAndIndexStream = sitemap_index_stream_1.SitemapAndIndexStream;
var sitemap_stream_1 = require("./lib/sitemap-stream");
exports.streamToPromise = sitemap_stream_1.streamToPromise;
exports.SitemapStream = sitemap_stream_1.SitemapStream;
__export(require("./lib/errors"));
__export(require("./lib/types"));
var utils_1 = require("./lib/utils");
exports.lineSeparatedURLsToSitemapOptions = utils_1.lineSeparatedURLsToSitemapOptions;
exports.mergeStreams = utils_1.mergeStreams;
exports.validateSMIOptions = utils_1.validateSMIOptions;
exports.normalizeURL = utils_1.normalizeURL;
exports.ReadlineStream = utils_1.ReadlineStream;
var xmllint_1 = require("./lib/xmllint");
exports.xmlLint = xmllint_1.xmlLint;
var sitemap_parser_1 = require("./lib/sitemap-parser");
exports.parseSitemap = sitemap_parser_1.parseSitemap;
exports.XMLToSitemapItemStream = sitemap_parser_1.XMLToSitemapItemStream;
exports.ObjectStreamToJSON = sitemap_parser_1.ObjectStreamToJSON;
